packages/docs/docs/medications/formulary.md

```diff
@@ -0,0 +1,382 @@
+  // Dispense 200g total
+  amount: {
+    value: 200,
+    unit: 'g',
+    system: 'http://unitsofmeasure.org',
+  },
+  // Each ingredient strength is listed as a ratio out of 100g
+  ingredient: [
+    {
+      isActive: true,
+      itemCodeableConcept: {
+        text: 'Baclofen powder',
+        coding: [
+          {
+            system: 'http://www.nlm.nih.gov/research/umls/rxnorm',
+            code: '1292',
+          },
+        ],
+      },
+      strength: {
+        numerator: {
+          value: 5,
+          unit: 'g',
+          system: 'http://unitsofmeasure.org',
+        },
+        denominator: {
+          value: 100,
+          unit: 'g',
+          system: 'http://unitsofmeasure.org',
+        },
+      },
+    },
+    {
+      isActive: true,
+      itemCodeableConcept: {
+        text: 'Ketoprofen powder',
+        coding: [
+          {
+            system: 'http://www.nlm.nih.gov/research/umls/rxnorm',
+            code: '6142',
+          },
+        ],
+      },
+      strength: {
+        numerator: {
+          value: 10,
+          unit: 'g',
+          system: 'http://unitsofmeasure.org',
+        },
+        denominator: {
+          value: 100,
+          unit: 'g',
+        },
+      },
+    },
+    {
+      isActive: true,
+      itemCodeableConcept: {
+        text: 'Capcaicin 0.75 mg/mL topical lotion',
+        coding: [
+          {
+            system: 'http://www.nlm.nih.gov/research/umls/rxnorm',
+            code: '1992',
+          },
+        ],
+      },
+      strength: {
+        numerator: {
+          value: 0.075,
+          unit: 'g',
+        },
+        denominator: {
+          value: 100,
+          unit: 'g',
+        },
+      },
+    },
+    {
+      isActive: true,
+      itemCodeableConcept: {
+        text: 'Tetracaine 10 mg/mL topical cream',
+        coding: [
+          {
+            system: 'http://www.nlm.nih.gov/research/umls/rxnorm',
+            code: '10391',
+          },
+        ],
+      },
+      strength: {
+        numerator: {
+          value: 2,
+          unit: 'g',
+        },
+        denominator: {
+          value: 100,
+          unit: 'g',
+        },
+      },
+    },
+    {
+      isActive: false,
+      itemCodeableConcept: {
+        text: 'PLO flowable Pluronic Lecithin Organogel',
+        coding: [
+          {
+            system: 'http://www.nlm.nih.gov/research/umls/rxnorm',
+            code: '012345',
+          },
+        ],
+      },
+      strength: {
+        numerator: {
+          value: 82.9,
+          unit: 'g',
+        },
+        denominator: {
+          value: 100,
+          unit: 'g',
+        },
+      },
+    },
+  ],
+  packaging: {
+    type: {
+      text: 'Tube',
+      coding: [
+        {
+          system: 'http://terminology.hl7.org/CodeSystem/medicationknowledge-package-type',
+          code: 'tube',
+          display: 'Tube',
+        },
+      ],
+    },
+  },
+//...
+}
+```
+
+</details>
+
```

Comment:
Consider adding a link to a bundle of this resource and prompt people download and upload to their account.